app/embeds/index.js

```diff
@@ -115,6 +116,13 @@ export default [
     component: Gist,
     matcher: matcher(Gist),
   },
+  {
+    title: "Diagrams",
```

Comment:
```suggestion
    title: "Diagrams.net",
```

Looks like "diagrams.net" is the name of the product

![image](https://user-images.githubusercontent.com/380914/130290952-593a2cde-9cbf-446e-a7cb-632181e7b5db.png)